jekyll/_cci2/vm-service.adoc

```diff
@@ -66,6 +68,18 @@ You will need to associate the `circleci` user with the image you want to use as
 
 To enable the use of these custom images for your installation, you will need to provide the individual AMI IDs in the Settings accessible from the Management console (for example, `<your-circleci-id>.com:8800`).
 
+=== Creating a Windows AMI
+_Introduced in CircleCI Server v2.18.3_
+
+Creating a Windows image and specifying it under the VM Service settings lets your users run jobs on dedicated Windows VMs. To create your Windows image run through the following steps. Windows images are built on CircleCI, so we suggest you run through this process once your installation is up and running. Alternatively you can use any other CircleCI account – including on our managed Cloud service – to generate the image:
+
+. Create a new repository under your GitHub or GitHub Enterprise account.
+. Copy the contents of **<INSERT WINDOWS IMAGE BUILDER REPO LINK>** into your new repo.
+. Open up your installation of CircleCI Server and https://circleci.com/docs/2.0/getting-started/#setting-up-your-build-on-circleci[connect your new repo] by clicking Follow Project from the Add Projects page.
+. Click Start Building to run the Windows image buider.
+. Your will find your new Windows AMI at the end of the `Summarize results` step in the job output.
```

Comment:
To be pedantic here, in the output you’ll find the AMI (Amazon Machine Image) ID, and you’ll need to be pasting the ID into the settings rather than the AMI itself.

Reply:
Ah thanks for this, I was wondering what the name of this was – ID – seems obvious now you've said it!

Reply:
Done